DataManagementSystem/Service/IRODSStorageElementHandler.py

```diff
@@ -440,7 +438,7 @@ def transfer_bulkFromClient( self, fileID, token, ignoredSize, fileHelper ):
     if not os.path.exists( dir_path ):
       return S_ERROR( 'Failed to receive data' )
     try:
-      os.chmod( dir_path, 0755 )
+      os.chmod( dir_path, stat.S_IRWXU | stat.S_IRGRP | stat.S_IXGRP | stat.S_IROTH | stat.S_IXOTH )
```

Comment:
Pylint complains about it. IIUC it complains because 
- it's not supported in python3
- it's not valid everywhere? 
Not sure though. Maybe 0o755 would have been enough, but what I set above is what syggested by python doc.